x-pack/legacy/plugins/siem/public/components/timeline/body/column_headers/column_header.tsx

```diff
@@ -34,69 +32,74 @@ const ColumnHeaderComponent: React.FC<ColumneHeaderProps> = ({
   draggableIndex,
   header,
   timelineId,
+  isDragging,
   onColumnRemoved,
   onColumnResized,
   onColumnSorted,
   onFilterChange,
   sort,
 }) => {
-  const [isDragging, setIsDragging] = React.useState(false);
-  const handleResizeStop: ResizeCallback = (e, direction, ref, delta) => {
-    onColumnResized({ columnId: header.id, delta: delta.width });
+  const resizableEnable = { right: true };
+  const resizableSize = {
```

Comment:
will this remain stable across renders?

Reply:
now it will, Thanks for pointing that out :)